qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,110 @@
+### Sparse vectors benchmark
+
+Performance results are publicly available for you to test. Qdrant's R&D developed a dedicated [open-source benchmarking tool](https://github.com/qdrant/sparse-vectors-benchmark) just to test sparse vector performance.
+
+A real-life simulation of sparse vector queries was run against the [NeurIPS 2023 dataset](https://big-ann-benchmarks.com/neurips23.html). All tests were done on an 8 CPU machine on Azure. 
+
+Latency (y-axis) has dropped significantly for queries. You can see the before/after here:
+
+![dropping latency](/blog/qdrant-1.8.x/benchmark.png)
+**Figure 1:** Dropping latency in sparse vector search queries across versions 1.7-1.8.
+
+The colors within the scatter plot show the frequency of the results. Frequency is highest where the red dots are. This tells us that latency for sparse vectors queries dropped by a factor of 16. Therefore, the time it takes to retrieve an answer with Qdrant is that much shorter. 
+
+This performance increase can have a dramatic effect on hybrid search implementations. [Read more about how to set this up.](https://qdrant.tech/articles/sparse-vectors/)
+
+FYI, sparse vectors were released in [Qdrant v.1.7.0](https://qdrant.tech/articles/qdrant-1.7.x/#sparse-vectors). They are stored using a different index, so first check out the documentation if you want to try an implementation.
+
+## CPU Resource Management
+
+Indexing is Qdrant’s most resource-intensive process. Now you can account for this by allocating compute use specifically to indexing. You can assign a number CPU resources towards indexing and leave the rest for search. As a result, indexes will build faster, and search quality will remain unaffected.
+
+This isn't mandatory, as Qdrant is by default tuned to strike the right balance between indexing and search. However, if you wish to define specific CPU usage, you will need to do so from `config.yaml`.
+
+This version introduces a `optimizer_cpu_budget` parameter to control the maximum number of CPUs used for indexing. 
+
+> Read more about `config.yaml` in the [Configuration document](https://qdrant.tech/documentation/guides/configuration/).
+
+```yaml
+    optimizer_cpu_budget: 0
+   # CPU budget, how many CPUs (threads) to allocate for an optimization job.
+```
+- If left at 0, Qdrant will keep 1 or more CPUs unallocated - depending on CPU size.
+- If the setting is positive, Qdrant will use this exact number of CPUs for indexing.
+- If the setting is negative, Qdrant will subtract this number of CPUs from the available CPUs for indexing.
+
+For most users, the default `optimizer_cpu_budget` setting will work well. We only recommend you use this if your indexing load is significant.
+
+![CPU management](/blog/qdrant-1.8.x/cpu-management.png)
+**Figure 2:** Allocating additional resources towards indexing.
+
+Our backend leverages dynamic CPU saturation to increase indexing speed. For that reason, there will be absolutely no impact on search query performance.
```

Comment:
> there will be absolutely no impact on search query performance.

This is wrong.

There will **always** be _some_ impact on search performance.

We better phrase it as: there will be minimal impact on search query performance

The whole idea is that we find a good balance between great indexing times and great search performance.